src/pages/Room/index.js

```diff
@@ -165,14 +152,12 @@ const Room = () => {
             {room.price.toFixed(2)} EUR
           </TotalPrice>
           <WrapperCheckout>
-            {loading ? <Loading /> : null}
-            <ErrorMessage>{error ? error : null}</ErrorMessage>
-            <PayloCheckout onClick={handleCheckout}>
-              <img
-                src={data.cashback_button_image || checkoutButton}
-                alt="checkout button"
-              />
-            </PayloCheckout>
+            {loading && <Loading />}
+            {error && <ErrorMessage>error</ErrorMessage>}
+            <CheckoutButton
+              handleClick={handleCheckout}
```

Comment:
just use onClick as previous I think it's ok. you don't need to reinventing new name if we have convention already which is onClick.

Reply:
fixed